src/udev.c

```diff
@@ -28,6 +28,17 @@
 
 #include "project.h"
 
+#define COMMUNICATIONS_CLASS            0x02
+#define WIRELESS_CLASS                  0xE0
+#define VENDOR_SPECIFIC_CLASS           0xFF
+#define ETHERNET_NETWORKING_SUBCLASS    0x06
+#define RADIO_FREQUENCY_SUBCLASS        0x01
+#define BLUETOOTH_PROTOCOL              0x01
```

Comment:
See classes.h.
If the existing table can't be used/extended, at least these `#define`s should be moved there